tests/Export/ProductServiceTest.php

```diff
@@ -421,4 +423,224 @@ public function testProductIsNotFoundIfMainVariantIsNotAvailableForSalesChannel(
 
         $this->assertEmpty($result->getElements());
     }
+
+    public function mainVariantDefaultConfigProvider(): array
+    {
+        $parentId = Uuid::randomHex();
+
+        return [
+            'export shopware default' => [
+                'config' => 'default',
+                'parentId' => $parentId,
+                'parentPrice' => 15,
+                'cheapest' => 2
+            ],
+            'export main parent' => [
+                'config' => 'parent',
+                'parentId' => $parentId,
+                'parentPrice' => 15,
+                'cheapest' => 2
+            ],
+            'export cheapest variant' => [
```

Comment:
How can this export the cheapest variant, when the `config` is set to `parent`? Also the whole data provider does not seem to have a single case for `cheapest` configuration. How are these cases covered?

Reply:
Ah yes, forgot to change the config. Will fix it.